index.html

```diff
@@ -1,38 +1,205 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <link rel="stylesheet" href="styles.css"/>
   <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    </head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="stylesheet.css"/>
+  <title>Portfolio</title>
+</head>
 <body>
-  <main class="Main-body">
-  
-    <header class="Portfolio-header">
+  <main class="container">
+      <header class="header_section">
       <nav>
-        <a href="#" target="_parent" class="portfolio-logo">My Portfolio</a>
-        <img src="./Images/Union-button.png" class="mobile-icon"/>
+        <a href="#" target="_parent" class="Portfolio_logo">My Portfolio</a>
+        <img src="./images/Union-button.png" class="mobile_menu_icon"/>
```

Comment:
- [ ] [OPTIONAL] I think it would be better to add a real Hamburger button to match the Figma design :+1: 

**Your Design**
![image](https://user-images.githubusercontent.com/102757126/198848802-9936b0cf-04d2-4ca4-9daa-2a255f121c2b.png)

**Figma Design**
![image](https://user-images.githubusercontent.com/102757126/198848831-a0df910c-fb6b-4d90-b80f-9b8fc85f5309.png)